ci-images/Dockerfile.subsystem

```diff
@@ -4,7 +4,6 @@ ENV VIRTUAL_ENV=/opt/venv
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 COPY --from=quay.io/openshift/origin-cli:latest /usr/bin/oc /usr/bin/kubectl /usr/bin/
-COPY --from=registry.k8s.io/kustomize/kustomize:v4.3.0 /app/kustomize /usr/bin/
```

Comment:
Would 5.6.0 work? I think 5.5.0 has been skipped for docker image release for some reason